.github/workflows/audit_build_verify.yml

```diff
@@ -3,20 +3,20 @@ name: Kwenta CI
 on:
   push:
   pull_request:
-    branches: [main, dev]
+    branches: [master, dev]
```

Comment:
Make sure to use our production branch `main`. We're not using the `master` branch.